### PR TITLE
refactor: schemas, contexts and migrations

### PR DESCRIPTION
**Pull request by ruilopesm**

### CODE REVIEW

File: lib/atomic/partnerships/partner.ex [hunk 1]
```diff
@@ -3,13 +3,12 @@ defmodule Atomic.Partnerships.Partner do
...
-  import Ecto.Changeset
+
   alias Atomic.Organizations.Organization
   alias Atomic.Uploaders
 
-  @required_fields ~w(name description organization_id)a
-
-  @optional_fields []
+  @required_fields ~w(name organization_id)a
+  @optional_fields ~w(description)a
```
Comment (MarioRodrigues10) [inline lib/atomic/partnerships/partner.ex]:
```suggestion
  @optional_fields ~w(description image)a
```

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
There will be some problems in activities seeds due to changes in field types , some make sure to merge #312  first and then solve those issues before i approve this.
The rest of the code is looking good, that's only thing stopping me from approving this.

File: lib/atomic/activities/activity.ex [hunk 2]
```diff
@@ -26,14 +25,10 @@ defmodule Atomic.Activities.Activity do
...
 
-  @doc false
   def changeset(activity, attrs) do
     activity
     |> cast(attrs, @required_fields ++ @optional_fields)
-    |> cast_assoc(:activity_sessions,
-      required: true,
-      with: &Session.changeset/2
-    )
+    |> cast_assoc(:activity_sessions, with: &Session.changeset/2)
```
Comment (ruilopesm) [inline lib/atomic/activities/activity.ex]:
```suggestion
    |> cast_assoc(:activity_sessions, required: true)
```

**Review by danielsp45 — DISMISSED** [review]

**Comment by RuiL1904** [comment]
Needs other PRs to be merged first 🙃

**Review by MarioRodrigues10 — APPROVED** [review]
What a great work :rocket:

**Pull request merged**